base/applications/cmdutils/taskkill/taskkill.c

```diff
@@ -34,12 +34,29 @@ static BOOL force_termination = FALSE;
 static WCHAR **task_list;
 static unsigned int task_count;
 
+static WCHAR opForceTerminate[] = L"f";
+static WCHAR opImage[] = L"im";
+static WCHAR opPID[] = L"pid";
+static WCHAR opHelp[] = L"?";
+static WCHAR opTerminateChildren[] = L"t";
+
+static PWCHAR opList[] = {opForceTerminate, opImage, opPID, opHelp, opTerminateChildren};
+
+#define OP_PARAM_INVALID -1
+
+#define OP_PARAM_FORCE_TERMINATE 0
+#define OP_PARAM_IMAGE 1
+#define OP_PARAM_PID 2
+#define OP_PARAM_HELP 3
+#define OP_PARAM_TERMINATE_CHILD 4
```

Comment:
You'd want an ifdef around this too.
You basically want your diff to have
* No red (-) lines
* Only green (+) lines that say `#ifndef __REACTOS__`/`#endif` (to disable upstream code) or are enclosed in `#ifdef __REACTOS__`/`#endif` (to add new code).

This will then make it easier for us to sync the code with the upstream source (you can copy over the updated file and restore everything marked `__REACTOS__`)